### PR TITLE
Update to v0.15.0

### PR DESCRIPTION
**Pull request by sigma-andex**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,9 +6,7 @@
   "devDependencies": {
     "github-release-notes": "^0.17.1",
     "paluh-litps": "^0.1.4",
-    "pulp": "^15.0.0",
-    "purescript": "^0.14.0",
-    "spago": "^0.19.1"
```
Comment (fsoikin) [inline package.json:11]: Hey @sigma-andex why did you remove these two dependencies from here?

**Comment by fsoikin** [comment]
@sigma-andex could you please also update `bower.json` with the new versions, so downstream libraries can have correct resolutions?

File: package.json (hunk 1 above)
Comment (sigma-andex) [reply package.json:11]: Updated

File: src/Data/Undefined/NoProblem.js [hunk 2]
```diff
@@ -1,3 +1,4 @@
-"use strict"
-
-exports.undefined = undefined;
+const undefinedImpl = function () {
+    return undefined
+}
```
Comment (fsoikin) [inline src/Data/Undefined/NoProblem.js]: Hey @sigma-andex, is this change on purpose or an oversight? If on purpose, what was the rationale?

```suggestion
const undefinedImpl = undefined
```

File: package.json [hunk 3]
```diff
@@ -6,9 +6,9 @@
...
+    "pulp": "^16.0.0",
+    "purescript": "^0.15.0",
+    "spago": "^0.20.8"
```
Comment (fsoikin) [inline package.json]:
```suggestion
    "purescript": "^0.15.2",
    "spago": "^0.20.9"
```

File: src/Data/Undefined/NoProblem.js (hunk 2 above)
Comment (sigma-andex) [reply src/Data/Undefined/NoProblem.js]: @fsoikin sorry I oversaw this comment. yes I guess an oversight

Comment (fsoikin) [reply src/Data/Undefined/NoProblem.js]: Great 👍 
Just wanted to validate my mental model.

**Comment by paluh** [comment]
Sorry for the delay. I'm going to do proper release during the weekend.

**Comment by paluh** [comment]
And of course thanks a lot for your work!
